apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -9936,10 +9946,10 @@ span#HW_badge_cont:nth-child(3) {
     .padding-top-fixed-mobile {
         padding-top: 90px;
     }
-    .page-has-tabs .padding-top-fixed-mobile,
-    .on-user-page .padding-top-fixed-mobile {
-        padding-top: 130px !important;
-    }
+    //.page-has-tabs .padding-top-fixed-mobile,
```

Comment:
if it will be no use in the future , you can remove it

Reply:
@shalinee-walkover not sure, can be used further.